### ISSUE TITLE
File does not exist 'src\xrstf\Composer52\ClassLoader.php' 

### ISSUE THREAD

**Issue by d3vit** [lobostome/FurryBear#5]
Trying to set this up on a local WAMP environment and I'm running into issues with this file. 

It's not included in the distro. Is there something I'm missing?

**Comment by lobostome** [comment]
Can you share:

1. the ```require``` section of your ```composer.json```
2. how you set up autoloading for your composer packages

**Comment by d3vit** [comment]
I'm not using composer which might have been the issue. Rolling into a custom WordPress plugin. I was able to work around this, though.

Appreciate the quick reply!

**Comment by d3vit** [comment]
I actually was able to pinpoint this issue. It was because the autoloader throws an error.

*spl_autoload_register* allows php to go through all registered autoloaders in order, but if you throw an uncaught error in the middle of that process it can't try the next autoloader.

This is what was happening to me. There was another class using an autoloader and *furrybear_autoload* was throwing an error. I edited line 87-91 of *SplClassLoader* to read:

    if (file_exists($f)) {
        require $f;
    }
